store/database/migrate/postgres/0002_create_table_instances_stageowner.up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS instances (
```

Comment:
These files should only contain new db changes. Like creating a table or amending a table to add a new column. Instance creation is already done in the previous change scipt

Reply:
Done

Reply:
I think you've misunderstood how the migration scripts work. The 1st script 001 has already been run. therefore you don't need to include the instance table as it was already created. If it's the first time you're running the app, then it will run the first and then the 2nd. 

If you need to make changes to the instance table it will be an AMEND. You wouldn't change the 1st script 001 but add a new one to amend the existing table.

Reply:
Done